static/src/js/providers/PanelWidthContextProvider/PanelWidthContextProvider.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useMemo } from 'react'
+import PropTypes from 'prop-types'
+import PanelWidthContext from '../../contexts/PanelWidthContext'
+
+const PanelWidthContextProvider = ({ children }) => {
+  const [panelsWidth, setPanelsWidth] = useState(500)
```

Comment:
I don't think this matters its getting updated immediately but ,the value over on the `this.width = 600 // The default width the panel is displayed when open`

Reply:
I think I'd rather use something obviously wrong. I tried it with 0 and it still works, so I'll do that, and add a comment